src/apps/Permissions/AssignPermissionPanel.js

```diff
@@ -153,11 +153,12 @@ class AssignPermissionPanel extends React.PureComponent {
           </Field>
 
           <EntitySelector
+            includeAnyEntity
             label="Grant permission to"
             labelCustomAddress="Grant permission to"
+            activeIndex={assignEntityIndex}
```

Comment:
This sorting looks peculiar: is it hardcoded/fixed items first and then items that receive external data (alphabetically)?
If it is so, do we try to sort this way all around? I was doing alphabetical sorting all around.

Reply:
Yeah, struggled with these props. We don't have a hard and fast rule for how we do this, but generally I prefer boolean props (e.g. `includeAnyEntity `) and then alphabetical sorting.

In this particular case though, the labels are a lot more important per-instance, since they describe what it's for and makes for easier reading, so I decided to keep them at the top 🤷‍♂️.